addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -127,6 +128,35 @@ export class Rtc extends Record {
     /** @type {import("@mail/static/libs/odoo_sfu/odoo_sfu").SfuClient} */
     sfuClient = undefined;
 
+    lastActions = {};
+    actionsStack = [];
+    lastSelfCallAction = undefined;
+
+    callActions = Record.attr([], {
+        compute() {
+            return callActionsRegistry
+                .getEntries()
+                .map(([key, action]) => [key, action.isActive(this.selfSession)]);
+        },
+        onUpdate() {
```

Comment:
Note that nothing awaits that items are added to registry. So the `callActions` can be incomplete.

Thankfully the code works because you observe `this.selfSession` in the compute, so usually this is recomputed when initiating a call, and usually the registry is full and ready at that time.

This is the 1st time I see a need for both a registry and have a trace of items in models. Could be worth adding some code to properly sync registry items in models in the future, so this is more reliable than now with lack of it :)